README.md

```diff
@@ -969,6 +970,14 @@ The information required to connect to Kafka will be automatically set for you b
 
 Before you begin fixing issues or adding new features to the Kogito Operator, see [Contributing to the Kogito Operator](docs/CONTRIBUTING.MD) and [Kogito Operator architecture](docs/ARCHITECTURE.MD).
 
+### Requirements
+
+- [Docker](https://www.docker.com/)
```

Comment:
AFAIK it should be possible to use also Podman instead of Docker for operator build.

Reply:
+1 in podman or buildah as as preferreable build tool

Reply:
Done. I had only set Docker because it's the one used by default.